experimental/packages/proto/src/metrics.ts

```diff
@@ -0,0 +1,156 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { ValueType } from '@opentelemetry/api-metrics';
+import { AggregatorKind, Histogram, MetricKind, MetricRecord, Point } from '@opentelemetry/sdk-metrics-base';
+import { RPCImpl } from 'protobufjs';
+import { hrTimeToLong, toAttributes } from './common';
+import { opentelemetry } from './generated';
+import { Fixed64 } from './types';
+
+export type MetricsClientOptions = {
+    rpcImpl: RPCImpl,
+    startTime: Fixed64,
+};
+export class MetricsServiceClient {
```

Comment:
Both `@grpc/grpc-js` and `grpc` need a special proto loader https://github.com/grpc/grpc-node/tree/master/packages/proto-loader and I don't find the service definition generated by protobufjs is compatible with the one of `@grpc/grpc-js` (without cumbersome setups).

Ideally I'd find the most compatible way is to convert proto files to json, so that for web we can populate protobuf object classes with [`protobuf.Root.fromJSON`](https://github.com/protobufjs/protobuf.js#using-json-descriptors), and for grpc with [`fromJSON`](https://github.com/grpc/grpc-node/blob/master/packages/proto-loader/src/index.ts#L395).

Reply:
The proto loader is only needed if you are loading proto definitions from json or from .proto files. The static code generated is actually capable of serializing without it and sending with gRPC. Yes, with some cumbersome setup, but this prevents us from having a dependency on grpc-js directly in this package which is not desireable for the http/json exporters.